resources/views/admin/datatableWithNew.blade.php

```diff
@@ -3,45 +3,50 @@
-				<div class="title is-7 has-text-centered">
-					@if($withEditLink ?? true)
-						<component :is="object.id ? 'a' : 'div'"
-								   :href="object.id ? `{{Request::url() }}/${object.id}` : '#'">
-							<span class="is-size-3" v-text="object.name || '{{ $createTitle }}'"></span>
-							<font-awesome-icon icon="link" v-if="object.id"></font-awesome-icon>
-						</component>
-					@else
-						<div>
+    <div>
+        @component('components.datatable')
+            @slot('deleteButton', true)
+            @slot('formattersData', $formattersData ?? null)
+            @slot('buttons')
+                @isset($fieldType)
+                    <a class="button is-light"
+                       href="{{ action('Admin\FieldController@index', $fieldType) }}">@lang('admin/kitchens.fields')</a>
+                @endisset
+                <button class="button is-light" @click="actions.newObjectForm">@lang('vue.add')</button>
+                @isset($buttons)
+                    @foreach($buttons as $button)
+                        {!! $button !!}
+                    @endforeach
+                @endisset
+            @endslot
+            <template slot-scope="{object, onUpdate}" v-if="object">
+                <div class="title is-7 has-text-centered">
+                    @if($withEditLink ?? true)
+                        <component :is="object.id ? 'a' : 'div'"
+                                   :href="object.id ? `{{Request::url() }}/${object.id}` : '#'">
+                            <span class="is-size-3" v-text="object.name || '{{ $createTitle }}'"></span>
+                            <font-awesome-icon icon="link" v-if="object.id"></font-awesome-icon>
+                        </component>
+                    @else
+                        <div>
 							<span class="is-size-3"
-								  v-text="object.name || object.name_{{\App::getLocale()}} ||'{{ $createTitle }}'"></span>
-						</div>
-					@endif
-				</div>
-				<dynamic-form :url="'{{Request::url() }}/edit' + (object.id ? `/${object.id}` : '')"
-							  :on-data-update="onUpdate"
-							  :method="object.id ? 'patch' : 'post'"
-				></dynamic-form>
-				@isset($extraSlotView)
-					@include($extraSlotView)
-				@endif
-			</template>
-		@endcomponent
-	</div>
+                                  v-text="object.name || object.name_{{\App::getLocale()}} ||'{{ $createTitle }}'"></span>
+                        </div>
+                    @endif
+                </div>
+                <dynamic-form :url="'{{Request::url() }}/edit' + (object.id ? `/${object.id}` : '')"
+                              :on-data-update="onUpdate"
+                              :method="object.id ? 'patch' : 'post'"
+                ></dynamic-form>
+                @isset($extraSlotView)
+                    @include($extraSlotView)
+                @endif
+            </template>
+        @endcomponent
+        @if($title == __('worker/worker.shifts'))
```

Comment:
maybe add another isset and include here? more general I think like:

`
@isset($extraSlotView)
	@include($extraSlotView)
@endif
`

also my version might need to be endisset - fix that too